test/functoria/test_cli.ml

```diff
@@ -32,25 +32,25 @@ let test_configure () =
   Alcotest.(check result_b)
     "configure"
     (`Ok
-      (Cli.Configure
-         {
-           depext = true;
-           extra_repo =
-             [
-               ( "opam-overlays",
-                 "https://github.com/dune-universe/opam-overlays.git" );
-               ( "mirage-overlays",
-                 "https://github.com/dune-universe/mirage-opam-overlays.git" );
-             ];
-           args =
-             {
-               context = (true, false);
-               output = None;
-               config_file = Fpath.v "config.ml";
-               context_file = None;
-               dry_run = false;
-             };
-         }))
+       (Cli.Configure
```

Comment:
having 7 spaces here is a bit weird - why is this needed?

Reply:
This is to make the indentation identical for polymorphic and normal variant arguments:

```ocaml
    (Ok
       (...))
    (`Ok
       (...))
```